### ISSUE TITLE
Adjust map pins to better deal with color blindness

### ISSUE THREAD

**Issue by GUI** [GUI/covid-vaccine-spotter#57]
I've heard from several users that the current map colors aren't great for red/green color blindness (which I know should have been obvious given the colors I picked :). I tried to mitigate this slightly by making the green ones bigger, but this could all use some better distinction and coloring to try and better deal with this (maybe just better colors, or also maybe consider different pin shapes).

**Comment by MoralCode** [comment]
adding icons may be a simple way to fix this that's sorta similar to the shape idea. for example,  a checkmark for "has availability", question mark  or dash (`-`) for unsure/no data, and maybe an X for "no available vaccines"

**Comment by GUI** [comment]
@MoralCode: I appreciate your thoughts on this! I ended up going with slightly different shapes in #104 instead of symbol. I think ideally I would have liked the symbol approach you suggested, but it seemed to get too busy with that many tiny symbol on an un-zoomed map, so I owned with different shapes instead. Perhaps a bit less intuitive, but between the shapes and tweaked colors, I'm hoping this is all still fairly obvious, while addressing the underlying issues. Thank you for the ideas and tagging the various other issues!
